pennylane_qiskit/converter.py

```diff
@@ -36,7 +36,7 @@
 import pennylane as qml
 import pennylane.ops as pennylane_ops
 from pennylane.tape.tape import rotations_and_diagonal_measurements
-from pennylane_qiskit.qiskit_device import QISKIT_OPERATION_MAP
+from pennylane_qiskit.qiskit_device_legacy import QISKIT_OPERATION_MAP
```

Comment:
since `QISKIT_OPERATION_MAP` isn't really a "legacy" object but will be used by the new device and conversion going forward, should we move it to either the new `qiskit_device` folder or to `converter`, and update imports accordingly?

Reply:
Hmm I personally lean towards moving it to converter (to me it seems more like a utility thing?) so that, suppose in 5 years we have a new new device, this import doesn't need to change. Plus it just makes sense intuitively to import the `QISKIT_OPERATION_MAP` from the file where it is actually defined.